src/Serilog.Extensions.Logging/Extensions/Logging/SerilogLoggerScope.cs

```diff
@@ -71,6 +83,10 @@ public void Enrich(LogEvent logEvent, ILogEventPropertyFactory propertyFactory)
                     logEvent.AddPropertyIfAbsent(property);
                 }
             }
+            else
+            {
+                scopeItem = propertyFactory.CreateProperty(NoName, _state).Value;
```

Comment:
It's a shame we don't have `CreatePropertyValue()` here, to skip the extra allocation. We could special-case `string` to improve it a little.